tests/integration/helpers.py

```diff
@@ -539,6 +539,8 @@ async def deploy_tempo_cluster(ops_test: OpsTest):
             status="active",
             timeout=2000,
             idle_period=30,
+            # FIXME: intermittent issue where tempo momentarily goes into error state
```

Comment:
I wouldn't leave this as a FIXME in this repo. If it really is an issue, we should open it on the tempo repository, and then link to it instead in this comment

Reply:
created the issue and linked it 
https://github.com/canonical/tempo-coordinator-k8s-operator/issues/90